docs/pages/sdk/core/asset.md

```diff
@@ -40,6 +52,6 @@ async lazymint(
 }>
 ```
 
-Each method in the `Asset` class returns a `Promise` that resolves to an object containing the `chain`, `collection`, and `token` of the minted asset.
+Each method in the `Asset` class returns a `Promise` that resolves to an object containing the `chain`, `collection`, and `token` of the minted asset. The `mintDrop` method returns a `Promise` that resolves to a `UUID` representing the ID of the drop that was minted.
```

Comment:
That should not, it should return the actual ID/IDs of the NFT(s) minted, which would make a lot more sense.